processor/pom.xml

```diff
@@ -92,6 +92,17 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <MapStruct-Generated-Comment>${project.groupId}:${project.artifactId}:${project.version}</MapStruct-Generated-Comment>
```

Comment:
> would suggest you finish this PR and leave the issue open, I'll just add my stuff along side with it and create a new PR then.

Ok.. 'Implementation-Version: 1.0.0-Beta3' it'll be ! I'll update and merge tonight.

Thanks for your input.